cmd/minishift/cmd/config/config.go

```diff
@@ -101,22 +101,22 @@ var (
-	SkipCheckIsoUrl                 = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true, nil)
-	WarnCheckIsoUrl                 = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true, false)
-	SkipCheckVMDriver               = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true, nil)
-	WarnCheckVMDriver               = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true, false)
-	SkipCheckVBoxInstalled          = createConfigSetting("skip-check-vbox-installed", SetBool, nil, nil, true, nil)
-	WarnCheckVBoxInstalled          = createConfigSetting("warn-check-vbox-installed", SetBool, nil, nil, true, false)
-	SkipCheckOpenShiftVersion       = createConfigSetting("skip-check-openshift-version", SetBool, nil, nil, true, nil)
-	WarnCheckOpenShiftVersion       = createConfigSetting("warn-check-openshift-version", SetBool, nil, nil, true, false)
-	SkipCheckOpenShiftGithubRelease = createConfigSetting("skip-check-openshift-github-release", SetBool, nil, nil, true, nil)
-	WarnCheckOpenShiftGithubRelease = createConfigSetting("warn-check-openshift-github-release", SetBool, nil, nil, true, false)
+	SkipCheckKVMDriver        = createConfigSetting("skip-check-kvm-driver", SetBool, nil, nil, true, nil)
+	WarnCheckKVMDriver        = createConfigSetting("warn-check-kvm-driver", SetBool, nil, nil, true, false)
+	SkipCheckXHyveDriver      = createConfigSetting("skip-check-xhyve-driver", SetBool, nil, nil, true, nil)
+	WarnCheckXHyveDriver      = createConfigSetting("warn-check-xhyve-driver", SetBool, nil, nil, true, false)
+	SkipCheckHyperVDriver     = createConfigSetting("skip-check-hyperv-driver", SetBool, nil, nil, true, nil)
+	WarnCheckHyperVDriver     = createConfigSetting("warn-check-hyperv-driver", SetBool, nil, nil, true, false)
+	SkipCheckIsoUrl           = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true, nil)
+	WarnCheckIsoUrl           = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true, false)
+	SkipCheckVMDriver         = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true, nil)
+	WarnCheckVMDriver         = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true, false)
+	SkipCheckVBoxInstalled    = createConfigSetting("skip-check-vbox-installed", SetBool, nil, nil, true, nil)
+	WarnCheckVBoxInstalled    = createConfigSetting("warn-check-vbox-installed", SetBool, nil, nil, true, false)
+	SkipCheckOpenShiftVersion = createConfigSetting("skip-check-openshift-version", SetBool, nil, nil, true, nil)
+	WarnCheckOpenShiftVersion = createConfigSetting("warn-check-openshift-version", SetBool, nil, nil, true, false)
+	SkipCheckOpenShiftRelease = createConfigSetting("skip-check-openshift-release", SetBool, nil, nil, true, nil)
+	WarnCheckOpenShiftRelease = createConfigSetting("warn-check-openshift-release", SetBool, nil, nil, true, false)
```

Comment:
This is better than `WarnCheckOpenShiftGithubReleas` or `SkipCheckOpenShiftGithubRelease`. @praveenkumar  WDYT? `WarnCheckOpenShiftRelease` or `SkipCheckOpenShiftRelease` can used for downstream too.

Reply:
👍